submitted_models/marble_hd2_sensor_config_3/specifications.md

```diff
@@ -0,0 +1,76 @@
+<!---This is a Markdown description of a robot model submitted for inclusion in the DARPA Subterranean Challenge Technology Repository -->
+
+# MARBLE HD2 Sensor Config 3
+This specifications.md file is a description and proof of virtual model validation for the MARBLE HD2 with Sensor Configuration 1. This robot may be launched using an ign launch command with the variable name `marble_hd2_sensor_config_3`.
+
+## Description
+This configuration is based on Superdroid HD2 ground robot. The marble sensor suite is located at the front of the HD2 chassis and includes stationary sensors (cameras, lidars, etc).  
+
+## Usage Instructions
+The robot can be used in the same manner as the MARBLE Husky robot.  
+
+## Usage Rights
+The same Rights are granted for the configuration as for the MARBLE Husky. No additional restrictions have to be taken into account for this configuration.
+
+### Cost and Scale
+The MARBLE HD2 has the following estimated commercial costs:
+* Base vehicle: $10,000
+* Sensor Suite: $14,000
+* Compute/support electronics: $4,000
+* Total: ~ $28,000
+
+Its weight is approximately 75 lbs (possible, but heavy to heft with a single person). 
+
+### Sensors
+This HD2 with sensor configuration 1 includes the following sensors. The specifications for these instruments are provided below in the [Validation Links](#validation_links) section.
+
+The following specific sensors are declared payloads of this vehicle:
+
+*D435i RGBD Camera, modeled by rgbd_camera plugin
+  -fixed, downward-facing at about 45 degrees (for examining terrain)
+*Ouster 3D Lidar (64 Channel), modeled by gpu_lidar plugin
```

Comment:
It looks like you're using two of these (horizontal and vertical). Can you make a note about the two lidars here?